packages/valory/skills/decision_maker_abci/rounds.py

```diff
@@ -127,7 +127,7 @@ class DecisionMakerAbciApp(AbciApp[Event]):
         },
         DecisionReceiveRound: {
             Event.DONE: BetPlacementRound,
-            Event.MECH_RESPONSE_ERROR: BlacklistingRound,
+            Event.MECH_RESPONSE_ERROR: DecisionReceiveRound,  # loop on the same state until Mech deliver is received
```

Comment:
What about the round timeout event?

Reply:
To my understanding, round timeout can be caused due to several other issues not only Mech not responding (for example, network latency, slow agent, etc.). What do you suggest to do in this cae?

Reply:
Isn't re-entering `DecisionReceiveRound` exactly what we are doing here?

Reply:
Yes, what I suggest is that we do the same in case of a round timeout event.